docs/user_guide/spin_system/spin_system.rst

```diff
@@ -59,14 +105,16 @@ tensor represented by a :ref:`sy_api` object.
 Since deuterium is a quadrupolar nucleus, :math:`I>1/2`, there also can be a quadrupolar coupling
 interaction between the nuclear quadrupole moment and the surrounding electric field gradient (EFG) tensor,
 defined in the optional *quadrupolar* key. An EFG tensor is a second-rank traceless
-symmetric tensor, and we describe its components with *Cq* and *eta*, i.e., the quadrupolar coupling constant
-and asymmetry parameter, respectively.  Additionally, we use the Euler angle orientations, *alpha*, *beta*,
-and *gamma*, which are the relative orientation of the EFG tensor from a common reference frame.
+symmetric tensor, and we describe its coupling to a quadrupolar nucleus with *Cq*
+and *eta*, i.e., the quadrupolar coupling constant and asymmetry parameter,
+respectively.  Additionally, we use the Euler angle orientations, *alpha*, *beta*,
+and *gamma*, which are the relative orientation of the EFG tensor from a common
+reference frame.
```

Comment:
Change the keyword names here and in above paragraph from italics into code text